docs/modules/ROOT/pages/cicd_concept.adoc

```diff
@@ -0,0 +1,40 @@
+= Continuous Delivery
+
+In order to ensure that code released to production is of the highest possible quality, we recommend
+setting up three separate deployment environments to play different roles in the Continuous Integration 
+/ Continuous Delivery process.
+
+== Target Enviroments
+
+A typical CI/CD framework would deploy images to the following three environments:
+
+=== Development
+
+The development environment is used to verify that features developed locally work in a
+cleanly set up environment, which may include changes to infrastructure such as
+endpoints, pipelines, back-end authentication, load tests, and database configuration.
+
+Merge requests should deploy so-called 
+https://about.gitlab.com/blog/2016/11/22/introducing-review-apps[Review-Apps]
+that allow developers and involved parties to verify feature implementation even before
+changes are merged into the main branch.
+
+Automated tests run by the pipeline are meant to make sure consistently that existing features
+don't break and manual testing is reduced to a minimum, if necessary at all.
+
+=== Integration
+
+The integration environment is used as a pre-production environment to confirm the correct
+functioning of all developed features. The images deployed here should be the actual images
+that are release candidates for production.
+
+=== Production
+
+The production environment should be self-explanatory: It's used as the live, productive system 
+accessed by real clients. Images that have been built, deployed, and approved on the integration 
+environment can be deployed here.
```

Comment:
I just noticed that a technical detail is missing here: It might be worthwhile to mention the Git tagging as a measure to create an audit trail, e.g.

> In extension to that, it's advisable to create an audit rail from the source code to the production state by tagging the deployed images with either the Git commit SHA or the related Git tag.

What do you think?

Reply:
Seems reasonable.  I am adding it.